webpack/assets/javascripts/react_app/components/common/DateTimePicker/date-time-picker.scss

```diff
@@ -1,6 +1,13 @@
+.bootstrap-datetimepicker-widget.dropdown-menu.top {
+  margin-top: -10px;
+}
+
+.bootstrap-datetimepicker-widget.dropdown-menu.bottom {
+  margin-top: 10px !important;
```

Comment:
We try not to use `!important` if possible,
maybe a small fix is needed in the css of PF core?

Reply:
I took it from comment of Maria, I have tried without beacause I don't like `!important`, didn't work though. And the code should die in next version, so I just don't want to invest too much of effort to it.